Added Charmion's Advice Link

luckythatsme opened the pull request:
Thank you Skillcrush!

redrambles commented:
Hey @luckythatsme !

This is a good start! We have a link to your advice page in the main index.html file (yay!) but no such `advice/charmion-lee-advice.html` file exists as part of this pull request. Perhaps it still needs to be added and committed? Checking git status will help you confirm.

Once you have added and committed your advice page, please push up your advice branch once more, which will automatically update this pull request. Then we should be all set to merge in your code.

Let us know if you have any questions!

Cheers,
Ann :)

redrambles commented:
Hey @luckythatsme !

Ok - we are getting closer!! A few things:

1) Your advice file is `charmion-lee-advice.html.html` and it's in the root of the project folder. Please rename it so that it only uses one '.html' extension, and move it into the 'advice' directory. 

2) In the index.html file, you accidentally removed someone else's link when you added your own, see here:
![](https://cldup.com/aORkLJVtP3.png)

This probably happened when you were resolving conflicts. No worries! Just add their link back in the file. You can copy/paste it if you look at [this view](https://github.com/skillcrush/git-practice/pull/489/files) and you scroll down to the index.html file (it's the line in red on the left).

Once that link is back in the index.html file and that your advice file has been renamed and moved to the advice directory, add all of your change in one go with `git add -A`, then commit as per usual and finally push up your `advice` branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code! Let us know if you have any questions!

Cheers,
Ann :)

luckythatsme commented:
@redrambles  I have updated my file name and followed your instructions, but got a conflict message. This same thing happened when I did this the first time, thus removing the other person's link to solve the conflict. So I guess I am confused on how to resolve it. I have updated the index. html and moved my advice file to the advice root folder. Please advise. Thank you! 
![Capture](https://user-images.githubusercontent.com/96362033/152583231-19aef18c-ea4d-4261-ad4d-24667ca308f5.JPG)

redrambles commented:
Hey @luckythatsme !

That's ok - a conflict is 100% normal in a group project, especially when everyone is editing the same file (the index.html file). In the code, find the conflict markers, then make sure to keep everyone's links in the highlighted sections (remove any duplicates if you see any), and then remove the conflict markers. 

Once you are good there, save the file, then add the index.html file once more, with your new conflict-resolving changes, and then commit as per usual.

Once that is complete, push up your advice branch once more, and we'll go from there! :)

Cheers,
Ann :)

luckythatsme commented:
@redrambles 
![Capture](https://user-images.githubusercontent.com/96362033/152615019-741b92cc-6d00-414b-b0de-5e276328581a.JPG)

twosavoie commented:
Hi @luckythatsme,

The next step would be to run `git pull origin advice`. This command will both fetch the changes on the remote origin repo and merge them into your local repo. There might be some conflicts - which is perfectly normal when multiple people are working on a project. 

In your code, find the conflict markers. They'll be around any conflicts that need to be resolved. Make sure to keep everyone's links in the highlighted sections and remove any duplicates you see. Then remove the conflict markers.

Once you are good there, save the file, then add the index.html file once more, with your new conflict-resolving changes, and then commit as per usual.

Once that is complete, push up your advice branch once more, and we'll go from there! :) 

🙂
Lisa

twosavoie commented:
Hey @luckythatsme,

We haven't heard from you in a while so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

xoxo,
Lisa
